openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSL.java

```diff
@@ -588,4 +620,41 @@ public static native void setCertificateBio(
      * Free x509 chain ({@code STACK_OF(X509)} pointer).
      */
     public static native void freeX509Chain(long x509Chain);
+
+    /**
+     * @see #setTlsExtStatusType(long, int)
+     */
+    public static final int TLSEXT_STATUSTYPE_ocsp = tlsExtStatusTypeOcsp();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^([a-z][a-zA-Z0-9]*|[A-Z][A-Z0-9_v]*)$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00115)